### PR TITLE
Add FaaS description in Portugese (#1207)

### LINKED ISSUE #1207: [Portuguese] Localize 'Function as a Service (FaaS)'
Localize [https://glossary.cncf.io/function-as-a-service/](https://glossary.cncf.io/function-as-a-service/)

### PR DESCRIPTION
**Pull request by pasimoes**
### Describe your changes

Created all content of FaaS Definition in PT-BR

### Related issue number or link (ex: `resolves #issue-number`)

resolves #1207 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by pasimoes** [comment]
Notes for Revision:
1- Please take care of comments in https://github.com/cncf/glossary/issues/1207#issuecomment-1372145269
2- CSS file came from the upstream (cncf/main) main repo. I just updated my fork.

**Comment by jihoon-seo** [comment]
@pasimoes Thank you for your contribution, but:
- please update this PR to contain the commit about pt-br version of FaaS, and to not contain the CSS update from `main` branch.
- desirably, please update your commit title and description like:
  - commit title: `Add FaaS description in Portugese`
  - commit description: `Signed-off-by: Paulo Alberto Simoes <p@simoes.me>`

**Comment by pasimoes** [comment]
Tomorrow I will fix this items. Thank you

File: content/pt-br/function-as-a-service.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: Função como um Serviço (FaaS)
+status: Completed
+category: Tecnologia
+tags: ["infraestrutura", "", ""]
+---
+
+## O que é
+
+Função como um Serviço, (FaaS - Function as a Service ), é um tipo de [serviço](/pt-be/serviço/) de [computação em nuvem](/pt-br/cloud-computing/) [sem servidor](/pt-br/serverless/) que permite a execução de código em resposta a eventos sem manter a complexa infraestrutura normalmente associado à criação e lançamento de aplicações de [microsserviços](/pt-br/microsserviços/).
```
Comment (edsoncelio) [inline content/pt-br/function-as-a-service.md:10]: Typo: `/pt-be/serviço/`

**Review by edsoncelio — CHANGES_REQUESTED** [review]
Thank you for your contribution @pasimoes!
Just a small typo and then LGTM :D

File: content/pt-br/function-as-a-service.md (hunk 1 above)
Comment (pasimoes) [reply content/pt-br/function-as-a-service.md:10]: fixed. tks

**Comment by pasimoes** [comment]
@edsoncelio and @MrErlison, please have a link to `microservices` pointing to the English description because I don't have it in PT-BR yet. We have to add a backlog.

**Comment by pasimoes** [comment]
I'd talked with @CathPag regarding the following suggestion, and she explained the decision to use samples from something other than proprietary products; also, to avoid maintenance cycles, the decision is not to point to specific open sources. So, I will remove it from my translation.


```
Alguns exemplos de FaaS em Nuvem Pública Gerenciado incluem:

- [AWS Lambda](https://aws.amazon.com/lambda/)
- [Funções do Microsoft Azure](https://azure.microsoft.com/en-us/services/functions/).
- [Funções do Google Cloud](https://cloud.google.com/functions/)
- [Funções do Oracle Cloud Infrastructure](https://www.oracle.com/cloud/cloud-native/functions/)

Alguns exemplos de Plataformas para execução de FaaS em Nuvem Pública e Privada:

- [OpenFaaS](https://www.openfaas.com)
- [Apache OpenWhisk](https://openwhisk.apache.org)
- [Knative](https://www.openfaas.com)
```

**Comment by edsoncelio** [comment]
Hi @pasimoes, the translation itself looks good, we just need to check the build errors; I didn't found the error 🤔

**Comment by MrErlison** [comment]
> Hi @pasimoes, the translation itself looks good, we just need to check the build errors; I didn't found the error 🤔

Hi @edsoncelio What do you think an empty commit to rebuild? Sometimes it works for me. 

`git commit --amend`

`git push --force `

**Comment by pasimoes** [comment]
My suggestion . I will close this PR. Clone the repo again made the translation and submit . It should fix any issue. Ok?

**Comment by MrErlison** [comment]
> My suggestion . I will close this PR. Clone the repo again made the translation and submit . It should fix any issue. Ok?

Try an empty commit first. It is easy way. I think.

**Comment by edsoncelio** [comment]
> > My suggestion . I will close this PR. Clone the repo again made the translation and submit . It should fix any issue. Ok?
> 
> Try an empty commit first. It is easy way. I think.

+1 Sorry for the later answer :'(
Try with a empty commit - If this not works you can open a new one (and I'll talk with the maintainers about this case)

**Comment by pasimoes** [comment]
This branch is broke. I will redo it.

**Pull request closed without merging**